frontend/src/pages/application_page/index.js

```diff
@@ -0,0 +1,196 @@
+    10: "How would you like to be remembered?",
+    11: "What is the most important lesson life has taught you?"
+  };
+
+  // FIXME: handle single question related to many roles (currently gets
+  //        printed under each rolename and all are overwritten)
+  const handleAnswerInput = (e, qID) => {
+    setAnswers({...answers, ...{[qID]: e.target.value}});
+  };
+
+  const toggleRole = (id) => {
+    if (rolesSelected.includes(id)) {
+      setRolesSelected(rolesSelected.filter((role) => role !== id));
+    } else {
+      setRolesSelected((existing) => [...existing, id]);
+    }
+  };
+
+  const onSubmit = () => {
+    // FIXME: make work with backend
+    //        ALSO need to create post submission page and then useNavigate() to it
+    if (rolesSelected.length) {
+      rolesSelected.forEach(role => {
+        const questionsIDs = roleQuestions[role];
+        const responses = questionsIDs.map(id => answers[id]);
+      })
+    } else {
+      alert("Submission failed, you must select at least one role to apply for!");
+    }
+  };
+
+  return (
+    <Container>
+      <CampaignImageCard sx={{boxShadow: 15}}>
+        <CardMedia
+          component="img"
+          height="100%"
+          image={DummyCampaignHeader}
+          alt={`campaign cover for ${1/*FIXME: this*/}`}
+        />
+      </CampaignImageCard>
+      <CampaignDescription>
+        <Typography>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+        </Typography>
+      </CampaignDescription>
+      <FormContainer style={{paddingLeft: "200px", paddingRight: "200px"}}>
+        <Section>
+          <SectionHeader>Applicant Details</SectionHeader>
+          {
+            loggedIn ?
+            <UserInfo>
+              <UserInfoFields>
+                <UserInfoTypography>Name:</UserInfoTypography>
+                <UserInfoTypography>zID:</UserInfoTypography>
+                <UserInfoTypography>Email:</UserInfoTypography>
+                <UserInfoTypography>Degree:</UserInfoTypography>
+              </UserInfoFields>
+              <div>
+                <UserInfoTypography><strong>{userInfo.name}</strong></UserInfoTypography>
+                <UserInfoTypography><strong>{userInfo.zid}</strong></UserInfoTypography>
+                <UserInfoTypography><strong>{userInfo.email}</strong></UserInfoTypography>
+                <UserInfoTypography><strong>{userInfo.degree}</strong></UserInfoTypography>
+              </div>
+            </UserInfo>
+            : <UserInfoTypography>
+              Please &nbsp;
```

Comment:
NOOOOOO just work on css instead of using these :)

otherwise if people were to highlight the this text and copy paste it could be a bit weird

Reply:
Please login or 

`<span>Please</span>` ====> style: margin-right: 2px